db/1.9/modules/ROOT/pages/database-connector-transactions.adoc

```diff
@@ -0,0 +1,78 @@
+= Configure Database Transactions Example - Mule 4
+
+Anypoint Connector for Database (Database Connector) enables you to execute database operations in the context of a transaction. Each operation has a Transactional Action field where you specify the type of joining action that operations can take regarding the active transaction.
+
+The following examples illustrates how to configure the Transactional Action field in Studio both for the Select operation, and when grouping operations using the Try scope component.
+
+== Configure the Transactional Action Field in Studio
+
+To configure the Transactional Action field for the Select operation follow the next steps:
+
+. In your Studio flow, select the *Select* operation.
+. In the connector configuration screen, click the *Advanced* tab.
+. Set the *Transactional action* field to any of the following options:
+
+* `ALWAYS_JOIN` +
+Expects a transaction to be in progress when a message is received. If there is no transaction, an error is raised.
+* `JOIN_IF_POSSIBLE` +
+Joins the current transaction if one is available. Otherwise, no transaction occurs.
+* `NOT_SUPPORTED` +
+Executes outside any existing transaction.
+
+The following screenshot shows the *Transactional action* configuration:
+
+.Transactional action configuration
+image::database-transactions-1.png[Transactional action configuration for the Select operation in Studio]
+
+In the XML editor, the `transactionalAction` configuration looks like this:
+[source,xml,linenums]
+----
+<db:select doc:name="Select" transactionalAction="ALWAYS_JOIN">
+			<db:sql ><![CDATA[SELECT * FROM PLANET WHERE name = :name]]></db:sql>
+</db:select>
+----
+
+== Configure the Transactional Action Field in the Try Scope to Group Operations
+
+Sometimes you need to execute several queries atomically in the context of the same transaction. If these queries are executed in the context of an already existing transaction, the queries belong to the same transaction.
+
+The following example shows that during a bank account transfer, you use two Database Connector Update operations to subtract money from one account and add it in another, but if any of the two operations fails, roll back both. If there’s no active transaction, you can start one by grouping the operations using the Try scope component:
+
+. In the *Mule Palette* view, search for `try` and select the *Try* component.
+. Drag the *Try* component onto the Studio canvas.
+. Drag an *Update* operation inside the *Try* component.
```

Comment:
"inside" or "onto"?
I think the latter: drag on top of rather than drag within....